src/main/java/Command.java

```diff
@@ -0,0 +1,19 @@
+abstract class Command {
+    private String action;
+    protected boolean isExit;
+
+    public Command(String action) {
+        this.action = action;
+        isExit = false;
+    }
+
+    public abstract void execute(TaskList tasks, Ui ui, Storage storage) throws CommandException;
+
+    public boolean isExit() {
```

Comment:
Should it be named "getExitStatus()" instead of "isExit()" ? The action of retrieving exit status may be more explicit this way :)

Reply:
Good point!